code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -400,11 +400,27 @@
 				informed = TRUE
 			switch(severity)
 				if(1)
-					L.receive_damage(0,10)
 					Stun(200)
+					if(!isrobotic(src))
+						L.receive_damage(0,10)
+						Stun(200)
+					else
+						// robotic species take far less damage, and roll to have their parts fly off on a heavy emp
+						// this is to stop emps outright killing them, and also because their organs have emp acts
+						L.receive_damage(0,4)
+						Stun(50)
+						if(!(L.body_part == CHEST || L.body_part == HEAD)) // only dismember legs/arms
+							if(prob(20))
```

Comment:
ambivalent on this. rng is pretty shitty especially not having it at all be modulated.

Reply:
would it be better to just do a prob(20) roll of removing one random limb so you can't lose multiple, or just remove the delimbing entirely